.docker/Dockerfile

```diff
@@ -0,0 +1,48 @@
+# This file is part of the Kimai time-tracking app.
+#
+# For the full copyright and license information, please view the LICENSE
+# file that was distributed with this source code.
+
+FROM php:7.2-fpm-alpine as kimai
+
+ARG VERSION
+ARG MAILER_FROM
+ARG MAILER_URL 
+
+RUN php -r "readfile('http://getcomposer.org/installer');" | php -- --install-dir=/usr/bin/ --filename=composer && \
+    apk add --no-cache \
+        bash \
+        git \
+        icu-dev \
+        freetype libpng libjpeg-turbo freetype-dev libpng-dev libjpeg-turbo-dev \
+        mysql-client \
+        unzip \
+        zip \
+        && \
+    docker-php-ext-install \
+        gd \
+        intl \
+        pdo_mysql \
+        zip \
+        && \
+    docker-php-ext-install \
+        gd \
+        intl \
+        pdo_mysql \
+        zip && \
+    mkdir -p /var/tmp/init-sql
+
+ADD https://github.com/kevinpapst/kimai2/archive/${VERSION}.tar.gz /var/tmp/kimai.tgz
+ADD env /var/tmp/kimai/.env
+
+WORKDIR /var/tmp/kimai
+RUN tar --strip-components=1 -zxvf /var/tmp/kimai.tgz && \
+    composer install --no-dev --optimize-autoloader && \
+    sed -i "s/kimai@example.com/${MAILER_FROM}/g" /var/tmp/kimai/.env && \
```

Comment:
I've done that for the dev image but the prod image should take it's settings from the compose file, that's the docker way.  If you think it would be cleaner we could use a env.template or something and use a place holder instead. e.g.

```MAILER_FROM={MAULER_FROM}```